ideascube/widgets.py

```diff
@@ -1,6 +1,56 @@
 from django.forms import widgets
 
 from ideascube.configuration import get_config
+from django.utils.html import mark_safe, escape
+from django.utils.encoding import force_text
+from django.forms.utils import flatatt
+
+
+_script_template = '''<script type="text/javascript">tinymce.init({{
+selector: "#{widget_id}",
+inline: true,
+menubar: {menubar},
+toolbar: {toolbar},
+plugins: "{plugins}",
+hidden_input: false
+}});</script>'''
+
+
+class RichTextEntry(widgets.Widget):
+    def __init__(
+        self, menubar=False, toolbar=False, plugins="autolink", attrs=None):
+        super().__init__(attrs=attrs)
+        self.menubar = menubar
+        self.toolbar = toolbar
```

Comment:
Will we be able to choose **for each entry** what buttons we want in the toolbar?